lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/widgets/guest_event_poap_offer_item.dart

```diff
@@ -101,22 +102,31 @@ class GuestEventPoapOfferItemState extends State<GuestEventPoapOfferItemView>
                         LemonRadius.small,
                       ),
                     ),
-                    child: ClipRRect(
-                      borderRadius: BorderRadius.circular(LemonRadius.xSmall),
-                      child: CachedNetworkImage(
-                        fit: BoxFit.cover,
-                        imageUrl: snapshot.data?.url ?? '',
-                        placeholder: (_, __) =>
-                            ImagePlaceholder.defaultPlaceholder(),
-                        errorWidget: (_, __, ___) =>
-                            ImagePlaceholder.defaultPlaceholder(),
-                      ),
+                    child: Stack(
+                      children: [
+                        Opacity(
+                          opacity: 0.5,
+                          child: ImageFiltered(
+                            imageFilter:
+                                ImageFilter.blur(sigmaX: 40, sigmaY: 40),
+                            child: CachedNetworkImage(
+                              fit: BoxFit.cover,
+                              imageUrl: snapshot.data?.url ?? '',
+                              placeholder: (_, __) =>
+                                  ImagePlaceholder.defaultPlaceholder(),
+                              errorWidget: (_, __, ___) =>
+                                  ImagePlaceholder.defaultPlaceholder(),
+                            ),
+                          ),
+                        ),
+                        const AppBackdrop(),
+                      ],
                     ),
                   ),
                 ),
               ),
             ),
-            const _Blur(),
+            // AppBackdrop(),
```

Comment:
not sure why we commented it here ?

Reply:
Ah, it's redundant, I will removed it later 👍